packages/react-native-web/src/vendor/react-native/NativeEventEmitter/RCTDeviceEventEmitter.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+import EventEmitter from '../emitter/EventEmitter';
+import EventSubscriptionVendor from '../emitter/EventSubscriptionVendor';
+
+import type EmitterSubscription from '../emitter/EmitterSubscription';
+
+function checkNativeEventModule(eventType: ?string) {
+  if (eventType) {
+    if (eventType.lastIndexOf('statusBar', 0) === 0) {
+      throw new Error(
+        '`' +
+          eventType +
+          '` event should be registered via the StatusBarIOS module',
```

Comment:
This check doesn't seem necessary for web

Reply:
The AppState one may be useful as the web module actually registers a listener.